packages/runtime/src/ctx.ts

```diff
@@ -131,6 +132,27 @@ export class Ctx implements Context {
 		}
 	}
 
+	/**
+	 * Asserts program length
+	 * @param approvalProgramBytes Approval program bytes
+	 * @param clearProgramBytes Clear program bytes
+	 * @param extraPages Extra pages. Default extra page is 0.
+	 */
```

Comment:
may be we can also add a limit here in description, like we have mentioned errors, so that while adding the method the user can read the docs and realise the limit

Reply:
No, it depends on the number of extra pages you have used. It's dynamic.
e.g extra page: 1, then limit is 4096
extra page: 2, then limit 6144

Reply:
so lets add the formula

Reply:
done